api/agent/grpc/runner.pb.go

```diff
@@ -594,6 +594,7 @@ type RunnerStatus struct {
 	KdumpsOnDisk         uint64   `protobuf:"varint,14,opt,name=kdumpsOnDisk,proto3" json:"kdumpsOnDisk,omitempty"`
 	SchedulerDuration    int64    `protobuf:"varint,15,opt,name=schedulerDuration,proto3" json:"schedulerDuration,omitempty"`
 	ExecutionDuration    int64    `protobuf:"varint,16,opt,name=executionDuration,proto3" json:"executionDuration,omitempty"`
+	IsNetworkDisabled    bool     `protobuf:"varint,17,opt,name=isNetworkDisabled,proto3" json:"isNetworkDisabled,omitempty"`
```

Comment:
i see this becoming something like `IsUserReady` or or `IsSpecialized` or something eventually, but protos aren't that hard to migrate so we can probably punt